feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -157,104 +168,119 @@ func TestQoSClassifierConfig(t *testing.T) {
 		termID      string
 		targetGroup string
 		dscpSet     []uint8
+		queueNumber string
```

Comment:
The other test cases appear to call this field "queueName". If it represents the same thing, use the same name.

Reply:
It is fixed as suggested